docs/source/user_guide_pipeline.rst

```diff
@@ -10,9 +10,10 @@ This page provides information about how to create a pipeline.
 
     Pipelines run asynchronously, see examples below.
 
-******************************
+
+*******************
 Creating Components
-******************************
```

Comment:
Is there a convention for the line length? I feel like we keep changing it from one PR to another :-)

Reply:
I think the convention from what I've seen is it needs to match the length of the text in the section